### ISSUE TITLE
reproducity problem

### ISSUE THREAD

**Issue by ZhengZixiang** [yangheng95/PyABSA#1]
On the Restaurant dataset, I can't reproduce the 0.86~0.87 accuracy result of the paper. I tried a variety of parameter combinations and only achieved the accuracy of 0.83. Do you have any other tricks or training strategies in the code implementation which are not mentioned in the paper and this repo?

**Comment by yangheng95** [comment]
Hello, sorry to see your question so late. I've just trained LCF-Bert on the restaurant dataset, with the built-in `train.py` and default parameters in the repository, the performance quickly achieving 0.85+. Can you post your parameter settings here?  Besides, LCF-Bert model defaults to deploy a single Bert layer to reduce memory consumption. To unleash the performance of LCF-Bert, see comments of line 34-35 in the lcf_bert.py file.

**Comment by yangheng95** [comment]
Hello, I have uploaded a training [log](./train.log) for the possible reproduction. Please let me know if there is any question.

**Comment by ZhengZixiang** [comment]
Thanks for providing training log file! Now I can reproduce the accuracy. I feel sorry for bringing the trouble.

The main reason is my training file different from yours at the beginning, in which I set the learning rate scheduler and adopted early-stop strategy. I forgot to change my code to your style. My original setting hurt the model performance to achieve the better accuracy.

I apologize again for opening this issue. Thank you for your answer and patience!
